models/engine/db_storage.py

```diff
@@ -76,7 +76,6 @@ def close(self):
         self.__session.remove()
 
     def get(self, cls, id):
-        """  retrieve one object """
```

Comment:
As good programming documentation it would be nice to identify params and what are you returning at each method of any class